modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP Workload Identity cluster, you must use the `ccoctl` binary to generate the cloud credentials manually and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Change the directory by running the following command:
++
+[source,terminal]
+----
+$ cd credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  generation: 1
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/dns.admin
```

Comment:
It is better to add a note for the role(s) used, like https://cert-manager.io/docs/configuration/acme/dns01/google/ noted:
```
Note: The use of the dns.admin role in this example role is 
for convenience. If you want to ensure cert-manager runs 
under a least privilege service account, you will need to 
create a custom role with the following permissions:
  dns.resourceRecordSets.*
  dns.changes.*
  dns.managedZones.list
```